cocos/ui/UIDeprecated.cpp

```diff
@@ -31,4 +31,4 @@ namespace ui {
     
 const Margin MarginZero =  Margin::ZERO ;
     
-}}
```

Comment:
NS_CC_BEGIN introduces `{`, so i think it is better to use `NS_CC_END` instead of `}` here.